app/src/main/java/social/entourage/android/entourage/category/EntourageCategoriesAdapter.kt

```diff
@@ -121,8 +123,10 @@ class EntourageCategoriesAdapter(
     }
 
     override fun getGroupView(groupPosition: Int, isExpanded: Boolean, convertView: View?, parent: ViewGroup): View {
-        var groupView = convertView
-        if (groupView == null) {
+        lateinit var groupView: View
```

Comment:
Do we need "lateinit" here

Reply:
You are right, I'll directly init the view with the right value